CRT.c

```diff
@@ -635,6 +643,8 @@ static int CRT_colorSchemes[LAST_COLORSCHEME][LAST_COLORELEMENT] = {
       [PRESSURE_STALL_THREEHUNDRED] = A_BOLD | ColorPair(Black, Blue),
       [PRESSURE_STALL_SIXTY] = A_NORMAL | ColorPair(White, Blue),
       [PRESSURE_STALL_TEN] = A_BOLD | ColorPair(White, Blue),
+      [FILE_DESCRIPTOR_USED] = A_BOLD | ColorPair(Green, Blue),
+      [FILE_DESCRIPTOR_MAX] = A_BOLD | ColorPair(Red, Blue),
```

Comment:
Is this an oversight or intentional?

Reply:
Intentional, as the background for the midnight commander theme is blue and not black. Having light blue on dark blue is harder to read than doing bright red there.